pcdsdevices/pim.py

```diff
@@ -572,14 +572,14 @@ class K2700(BaseInterface, Device):
               doc='Identity (name) of this device')
     reading = Cpt(EpicsSignalRO, ":Reading", kind="normal",
                   doc='Trigger and return a new measurement')
-    dcv = Cpt(EpicsSignalRO, ":GetDCV", kind="normal",
-              doc='DC voltage')
-    acv = Cpt(EpicsSignalRO, ":GetACV", kind="normal",
-              doc='AC voltage')
-    dci = Cpt(EpicsSignalRO, ":GetDCI", kind="normal",
-              doc='DC current')
-    aci = Cpt(EpicsSignalRO, ":GetACI", kind="normal",
-              doc='AC current')
+    dcv_range = Cpt(EpicsSignalRO, ":GetDCV", kind="normal",
```

Comment:
So this is technically a sort of "API Break", since we're changing the name of a component on this device.  If there are older scripts that expect `K2700.dcv`, they won't be able to find that name any longer. 

Practically, if you **know** you're the only one using this device and you won't break others' usage of the device, this is fine.  I just want to make sure this is the case.  

If it were me I'd update the documentation alone, but that may be out of an abundance of caution

Reply:
Thanks for the clarification, release notes have been updated to note the API breaks. I recently added this device class and there aren't any happi entries using it other than one I made so it should be safe to change the component names.